application/database/migrations/2023_03_30_143724_create_classifier_value_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use App\Enums\ClassifierValueType;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('classifier_values', function (Blueprint $table) {
+            $table->uuid('id')->primary()->default(DB::raw('gen_random_uuid()'));
+            $table->timestamps();
+            $table->softDeletesTz();
+            $table->enum('type', ClassifierValueType::values());
+            $table->string('value');
+            $table->string('name');
+            $table->json('meta')->nullable();
+            $table->unique(['type', 'value']);
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
+    {
+        Schema::dropIfExists('classifier_value');
```

Comment:
Looks like a typo. Consequently, `php artisan migrate:refresh` currently fails.

```suggestion
        Schema::dropIfExists('classifier_values');
```